doc/tutorial/alphabet/alphabet_gc_content.cpp

```diff
@@ -0,0 +1,52 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+//! [exercise]
+#include <array>     // std::array
+#include <iostream>  // std::cerr, std::endl
+#include <cstring>   // std::strlen
+#include <vector>    // std::vector
+
+#include <seqan3/alphabet/all.hpp>
+#include <seqan3/io/stream/debug_stream.hpp>
+
+using namespace seqan3;
+
+int main (int argc, char * argv[])
+{
+    // Test for the presence of an input sequence.
+    if (argc != 2)
+    {
+        std::cerr << "Usage: " << argv[0] << " <sequence>" << std::endl;
+        return 0;
+    }
+
+    // Get the length of the input.
+    size_t len = std::strlen(argv[1]);
```

Comment:
Please do not use cstring and strlen.

Reply:
Wait, no. Its more important to be consistent. Either free functions everywhere or member functions everywhere.

Reply:
Okay, then it will be member functions, as the whole tutorial uses them (and I currently cannot use global functions in the custom alphabet implementation).